embedded/devices/README.md

```diff
@@ -31,3 +31,23 @@ int set_state(int pin, uint8_t value) // Sets a specific pin a value (0 or 1)
 ### Corresponding Tests and Examples
 
 TODO...
+
+
+## CANDevices (RMS)
```

Comment:
As we noted, this is our chance to drop the RMS acronym for good, lets discuss this on slack.